### PR TITLE
Update clause_1.adoc

### PR DESCRIPTION
**Pull request by FransKnibbe**
Add section about Linked Data, add invitation to review and collaborate.

### CODE REVIEW

**Comment by FransKnibbe** [comment]
Extension of the introduction, adding a bit about Linked Data, aligning the introduction with the three chapters, and adding that the goal of the white paper is invoking involvement by potential users and investment by the OGC.

File: white_paper/clause_1.adoc [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 == Semantic Technologies
 
-Semantic technologies, in the scope of this document, relate to the use of formal ontologies to describe concepts: the relationships between "things" and categories of "things". There are a number of goals for using these technologies, which often relate to providing the capability to reason over the data and simplified dealing with heterogeneous data sources. Example goals include enabling machine readability, and federated searching.
+Semantic technologies, in the scope of this document, relate to the use of formal ontologies to describe concepts: the relationships between "things" and categories of "things". There are a number of goals for using these technologies, which often relate to providing the capability to reason over the data and simplified dealing with heterogeneous data sources. Example goals include enabling machine readability, and federated searching. A major area in which semantic technologies are uses is the Semantic Web, the part of the World Wide Web dealing with semantics.
```
Comment (jabhay) [inline white_paper/clause_1.adoc]:
```suggestion
Semantic technologies, in the scope of this document, relate to the use of formal ontologies to describe concepts: the relationships between "things" and categories of "things". There are a number of goals for using these technologies, which often relate to providing the capability to reason over the data and simplified dealing with heterogeneous data sources. Example goals include enabling machine readability, and federated searching. A major area in which semantic technologies are used is the Semantic Web, the part of the World Wide Web dealing with semantics.
```

**Review by jabhay — CHANGES_REQUESTED** [review]
Looks good. Just the one suggestion prior to acceptance. P.S. you've really got a knack for words. Great writing!

File: white_paper/clause_1.adoc [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+The spatial aspect of data is a core data dimension. Spatial information provides the capability to answer the question "where", providing additional context to whatever it relates to. The authors of this White Paper believe that semantic and graph technologies, together with  strategies for Linked Data, will increase the value that can be extracted from (geo)spatial data.
 
-This paper has two purposes. Firstly, to describe the benefits of representing geospatial data using semantics and graph technologies. Secondly, to outline some shortcomings of the existing GeoSPARQL implementation specification that, if addressed, would unlock its potential to a greater extent.
+Although many current and past work within the OGC involves semantic and graph technologies, there is one main applicable OGC standard: GeoSPARQL. GeoSPARQL offers both semantics for representing geospatial data and it specifies how geospatial data that are expressed using the Resource Description Framework (RDF) can be queried using SPARQL, the query language for RDF.
```
Comment (TallTed) [inline white_paper/clause_1.adoc:6]:
```suggestion
Although much current and past work within the OGC involves semantic and graph technologies, there is only one applicable OGC standard: GeoSPARQL. GeoSPARQL both offers semantics for representing geospatial data, and specifies how geospatial data that are expressed using the Resource Description Framework (RDF) can be queried using SPARQL, the query language for RDF. 
```

**Review by jabhay — APPROVED** [review]

**Pull request merged**